src/test/java/seedu/address/model/AddressBookTest.java

```diff
@@ -76,6 +77,39 @@ public void hasTag_tagWithSameIdentityFieldsInAddressBook_returnsTrue() {
         assertTrue(addressBook.hasTag(editedCS2103));
     }
 
+
+    @Test
+    public void setTag_tagInAddressBook_returnsTrue() {
+        AddressBook dummyAddressBook = addressBook;
```

Comment:
This does not create a new addressbook, so the same addressbook is used through out all testcases. Is this intended?

Reply:
Yes I just want to test the behaviour of the setTag.. there's no intention to create new AddressBook

Reply:
As in the addressbook will be reused so all the tags that you have added previously will persist across all testcases.

Reply:
Maybe can set up an empty addressbook by using `@beforeEach`.